.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
+.env.template
+.vscode
+.gitignore
+README.md
```

Comment:
This list is reasonable - you've included the most important file, `.env`. But some more files/folders could be added. E.g.: `.vagrant`, `Vagrantfile`, `__pycache__`, `.git`, `Dockerfile`